build.gradle

```
@@ -77,7 +77,7 @@ buildscript {
             "json"                        : "org.json:json:20220320",
 
             // PayPal
-            "nativeCheckout"              : "com.paypal.checkout:android-sdk:1.2.1",
+            "nativeCheckout"              : "com.paypal.checkout:android-sdk:1.3.2",
```

qq here - should this be version 1.3.0, that's the latest release i'm seeing [here](https://github.paypal.com/nativexo/android-native-checkout/releases/tag/external-v1.3.0)
```suggestion
            "nativeCheckout"              : "com.paypal.checkout:android-sdk:1.3.0",
```

---

hmmm.. great catch.
I feel the latest version might not have been tagged in the releases.

@tdchow please correct me if I'm wrong, but I believe 1.3.2 is the latest that we had to consume, correct?

---

Yep, it looks like the latest versions have not been tagged in the GitHub releases. The correct version to use is 1.3.2.

https://repo1.maven.org/maven2/com/paypal/checkout/android-sdk/1.3.2/